cmd/gotypeconv/main.go

```diff
@@ -21,6 +21,12 @@ import (
 	"golang.org/x/tools/go/loader"
 )
 
+var unused = 1
+
+var NoComment = 1
+
+var _ = fmt.Sprintf("", 1, 2, 3)
```

Comment:
**[govet]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
no formatting directive in Sprintf call